qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -421,32 +477,42 @@
             get_log_cmd = cat /var/log/qemu-ga/qemu-ga.log |grep "read data"
             tmp_file = "/tmp/qga_test"
             Windows:
+                check_vioser = yes
                 get_log_cmd = type c:\qga.log |findstr /c:"read data"
                 tmp_file = "C:\qga_test.txt"
         - gagent_query_chardev:
             no isa_serial
             gagent_check_type = query_chardev
+            Windows:
+                check_vioser = yes
         - gagent_check_after_init:
             no Windows
             gagent_check_type = after_init
         - gagent_frozen_io:
             only Windows
+            check_vioser = yes
             gagent_check_type = frozen_io
             iozone_cmd = "start /b for /l %i in (1,1,1000) do "
             iozone_cmd += "WIN_UTILS:\Iozone\iozone.exe -azR -r 64k -s 1G -M -i 0 -i 1 -b iozone.xls"
             iozone_cmd += " -f C:\testfile > C:\frozen_io_log.txt"
         - gagent_vss_status:
             only Windows
+            check_vioser = yes
             gagent_check_type = vss_status
         - gagent_with_migrate:
             only virtio_serial
             gagent_check_type = with_migrate
             mig_speed = 512M
+            Windows:
+                check_vioser = yes
         - gagent_user_logoff:
             only Windows
+            check_vioser = yes
             gagent_check_type = user_logoff
         - gagent_qgastatus_after_remove_qga:
             gagent_check_type = qgastatus_after_remove_qga
+            Windows:
+                check_vioser = yes
```

Comment:
The above `check_vioser = yes` specification for test cases seems not necessary.